app/components/common/userInput/index.tsx

```diff
@@ -0,0 +1,31 @@
+interface UserInputProps {
+  label: string;
+  placeholder: string;
+  link?: string;
+}
+
+const UserInput: React.FC<UserInputProps> = ({ label, placeholder, link }) => (
+  <div className="mb-6">
+    <p className="text-sm text-black mb-2">{label}</p>
+    <div className={`${link ? 'flex' : ''}`} data-testid="user-input-wrapper">
```

Comment:
Is this check needed? Does having the `flex` property always create styling issues ?

Reply:
yeah the check isn't needed for the current use case, i am removing it for now